script.js

```diff
@@ -35,3 +35,216 @@ function scrollToSectionAbout() {
 }
 
 scrollToContact.addEventListener('click', scrollToSectionAbout);
+
+const project1Array = [
+  {
+    title: 'Multi-Post Stories',
+    image: ['images/home/Placeholder.svg', './images/home/projects-dialog.png'],
+    description: [
+      'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.',
+      'Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore, repellendus voluptas consequatur officiis quae quos vel dolorem deserunt ab magni, vero, quo aliquid et eum optio illum. Animi nemo aspernatur aperiam iusto quisquam eaque voluptatum optio, tempore sint expedita nulla deleniti consectetur repellendus numquam cumque autem saepe libero harum ab!',
+    ],
+    technologies: ['css', 'html', 'bootstrap', 'Ruby'],
+  },
```

Comment:
- [ ] Please each object contains at least the following information about the project: name(title), description, featured image, technologies, link to live version, and link to the project source code. kindly add "link to live version", and "link to the project source code" in your array.
- [ ] Just a reminder: Modal popup data is populated with the data from the projects array